### PR TITLE
fix: Set icon fill to currentcolor

### PR DESCRIPTION
**Pull request by AmarTrebinjac**
## Changes

Update icon fill because it had a different color in light mode

## Events

Did you introduce any new tracking events?

## Experiment

Did you introduce any new experiments?

## Manual Testing

> [!CAUTION]
> Please make sure existing components are not breaking/affected by this PR

### Preview domain
https://mi-679.preview.app.daily.dev

### CODE REVIEW

File: packages/shared/src/components/icons/Folder/outlined.svg [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (omBratteng) [inline packages/shared/src/components/icons/Folder/outlined.svg:1]:
```suggestion
<svg width="32" height="32" viewBox="0 0 32 32" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
```

File: packages/shared/src/components/icons/Folder/outlined.svg [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
+<path id="Union" fill-rule="evenodd" clip-rule="evenodd" d="M4.33325 8C4.33325 6.71134 5.37792 5.66667 6.66658 5.66667H14.2686C14.8811 5.66667 15.469 5.90748 15.9055 6.33714L17.5947 8H23.9999C26.025 8 27.6666 9.64162 27.6666 11.6667V22.6667C27.6666 24.6917 26.025 26.3333 23.9999 26.3333H7.99992C5.97487 26.3333 4.33325 24.6917 4.33325 22.6667V8ZM6.66658 7.66667C6.48249 7.66667 6.33325 7.8159 6.33325 8V22.6667C6.33325 23.5871 7.07944 24.3333 7.99992 24.3333H23.9999C24.9204 24.3333 25.6666 23.5871 25.6666 22.6667V11.6667C25.6666 10.7462 24.9204 10 23.9999 10H17.1851C16.9226 10 16.6706 9.89679 16.4836 9.71265L14.5024 7.76245C14.4401 7.70107 14.3561 7.66667 14.2686 7.66667H6.66658Z" fill="currentcolor"/>
```
Comment (omBratteng) [inline packages/shared/src/components/icons/Folder/outlined.svg:3]:
```suggestion
<path id="Union" fill-rule="evenodd" clip-rule="evenodd" d="M4.33325 8C4.33325 6.71134 5.37792 5.66667 6.66658 5.66667H14.2686C14.8811 5.66667 15.469 5.90748 15.9055 6.33714L17.5947 8H23.9999C26.025 8 27.6666 9.64162 27.6666 11.6667V22.6667C27.6666 24.6917 26.025 26.3333 23.9999 26.3333H7.99992C5.97487 26.3333 4.33325 24.6917 4.33325 22.6667V8ZM6.66658 7.66667C6.48249 7.66667 6.33325 7.8159 6.33325 8V22.6667C6.33325 23.5871 7.07944 24.3333 7.99992 24.3333H23.9999C24.9204 24.3333 25.6666 23.5871 25.6666 22.6667V11.6667C25.6666 10.7462 24.9204 10 23.9999 10H17.1851C16.9226 10 16.6706 9.89679 16.4836 9.71265L14.5024 7.76245C14.4401 7.70107 14.3561 7.66667 14.2686 7.66667H6.66658Z" />
```

File: packages/shared/src/components/icons/Folder/outlined.svg [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
 <g id="&#60;Icon&#62;only">
```
Comment (omBratteng) [inline packages/shared/src/components/icons/Folder/outlined.svg:2]:
```suggestion
```

File: packages/shared/src/components/icons/Folder/outlined.svg [hunk 4]
```diff
@@ -1,5 +1,5 @@
...
+<path id="Union" fill-rule="evenodd" clip-rule="evenodd" d="M4.33325 8C4.33325 6.71134 5.37792 5.66667 6.66658 5.66667H14.2686C14.8811 5.66667 15.469 5.90748 15.9055 6.33714L17.5947 8H23.9999C26.025 8 27.6666 9.64162 27.6666 11.6667V22.6667C27.6666 24.6917 26.025 26.3333 23.9999 26.3333H7.99992C5.97487 26.3333 4.33325 24.6917 4.33325 22.6667V8ZM6.66658 7.66667C6.48249 7.66667 6.33325 7.8159 6.33325 8V22.6667C6.33325 23.5871 7.07944 24.3333 7.99992 24.3333H23.9999C24.9204 24.3333 25.6666 23.5871 25.6666 22.6667V11.6667C25.6666 10.7462 24.9204 10 23.9999 10H17.1851C16.9226 10 16.6706 9.89679 16.4836 9.71265L14.5024 7.76245C14.4401 7.70107 14.3561 7.66667 14.2686 7.66667H6.66658Z" fill="currentcolor"/>
 </g>
```
Comment (omBratteng) [inline packages/shared/src/components/icons/Folder/outlined.svg:4]:
```suggestion
```

**Comment by AmarTrebinjac** [comment]
@omBratteng I am going to be honest, brother, I have no idea what I'm looking at with all those suggestions 😂 

Why do we need to change all this?

**Review by omBratteng — APPROVED** [review]

**Pull request merged**